### ISSUE TITLE
getting a forbidden on this rss feed for some reason? when entering feed in browser am able to access with out issues?

### ISSUE THREAD

**Issue by mhussaincov93** [gpodder/gpodder#1175]
hi,
this feed:
https://petri.com/feed/mjfchat
is giveing me a forbidden status when updateing or trying to add to gpodder.
when accessed in the brave browser all works?
could it be a user agent issue at play?
on a related note,
would it be possible to have a setting where you are able to change the user agent for cases like these?
Majid

**Comment by mhussaincov93** [comment]
hi,

changed user agent in the file you mentioned,

no dice,

I don't understand what the issue could be?

here's how I set my useragent string etc,

note, I commented out the useragent that was present by default using

the #

char.

User-Agent string for downloads
#user_agent = 'gPodder/%s (+%s) %s' % (__version__, __url__,
platform.system())
user_agent = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML,
like Gecko) Chrome/95.0.4638.40 Safari/537.36"
note, I used quotes to encapsulate the string.

i'm no programmer, but that aught to work write?

I assume that using the # char will make gpodder ignore the default string?

Majid

**Comment by auouymous** [comment]
That is correct and should be using the new user agent. Are you using a new gpodder version, new python version and is your other software updated?

**Comment by mhussaincov93** [comment]
hi,

i'm running on archlinux,

python version is 3.9.7

gpodder version is

3.10.21

I believe, it's the latest from the archlinux repos

would it be better if I ran from git?

there is a gpodder-git package in the aur.

on a related question kinda,

when gpodder checks for updates,

how long does it wate between requests to the servers of the podcast feeds?

is it a barrage of requests at once? or a few seconds between every request?

reason me asking is I have 165 podcasts and most of them are from one
location.

if it is possible could a config setting be added where you can set how
long gpodder wates between every request when checking for updates.

if that does not make sence think of it like in wget terms

-w = seconds

how long the programme will wate between every http request to the server.

also don't want my ip being banned for barraging the servers.

**Comment by elelay** [comment]
To see if it's the many requests triggering the failure,try adding to [feedcore.py line 180](https://github.com/gpodder/gpodder/blob/3.10.21/src/gpodder/feedcore.py#L180)
```
        import time
        time.sleep(5)
````
(just the line before)
````
        stream = util.urlopen(url, headers)
```
as you have guessed it waits 5s before getting each feed from the server...

**Comment by mhussaincov93** [comment]
my god!

i'm so sorry for my late responce!

that code snip worked a charm!

would it be possible to have this as a togglable setting?

Majid
